### ISSUE TITLE
🐛 Bug Report: Cannot deploy collections in appwrite.json with Int Attributes

### ISSUE THREAD

**Issue by maximilianmaihoefner** [appwrite/appwrite#2894]
### 👟 Reproduction steps

1. Create a new Integer Attribute via the Web Interface, enter a name and leave all other fields empty and click create.
2. Connect the Appwrite CLI 2.0 using `appwrite login`
3. Create `appwrite.json` using `appwrite init --all`
4. Inspect the Integer Attribute in `appwrite.json`, it should show a min/max value of `-9223372036854776000`/`9223372036854776000`.
5. Try to deploy the `appwrite.json` using `appwrite deploy collection`
6. Choose the collection and confirm the overwrite.
7. An Error gets thrown.

### 👍 Expected behavior

Should be able to apply the created `appwrite.json`.

It would also be nice if the error message could specify the allowed min/max values, or a bit more description why the passed value is invalid.

### 👎 Actual Behavior

$ appwrite deploy collection
? Which collections would you like to deploy? MyCollection (62212c12df87ffcd4478)
ℹ Info Deploying collection MyCollection ( 62212c12df87ffcd4478 )
ℹ Info Collection MyCollection ( 62212c12df87ffcd4478 ) already exists.
? Are you sure you want to override this collection? This can lead to loss of data! Type "YES" to confirm. YES
ℹ Info Updating attributes ... 
✗ Error Invalid min: Value must be a valid integer

### 🎲 Appwrite version

Different version (specify in environment)

### 💻 Operating system

MacOS

### 🧱 Your Environment

Using Appwrite 0.13.0 but that isn't available in the 'Appwrite version'-Dropdown yet.

The Server was upgraded from 0.12.1 using the instructions at: https://appwrite.io/docs/upgrade running the Install and Data Migration commands.

### 👀 Have you spent some time to check if this issue has been raised before?

- [X] I checked and didn't find similar issue

### 🏢 Have you read the Code of Conduct?

- [X] I have read the [Code of Conduct](https://github.com/appwrite/appwrite/blob/HEAD/CODE_OF_CONDUCT.md)

**Comment by maximilianmaihoefner** [comment]
This is the full `appwrite.json`-File created by the CLI:
```
{
    "projectId": "test",
    "projectName": "Test",
    "collections": [
        {
            "$id": "62212c12df87ffcd4478",
            "$read": [],
            "$write": [],
            "name": "MyCollection",
            "enabled": true,
            "permission": "collection",
            "attributes": [
                {
                    "key": "myIntAttribute",
                    "type": "integer",
                    "status": "available",
                    "required": false,
                    "array": false,
                    "min": -9223372036854776000,
                    "max": 9223372036854776000,
                    "default": null
                }
            ],
            "indexes": []
        }
    ]
}
```

**Comment by christyjacob4** [comment]
@maximilianmaihoefner  Thank you for reporting this. We'll look into it 🙂

**Comment by stnguyen90** [comment]
FYI, min value is -9223372036854775808 and max value is 9223372036854775807.

**Comment by Meldiron** [comment]
I can confirm this problem is on the server side, not in the SDK. Same values are provided in Appwrite Console and the error above cones from axios (HTTP client) error.

**Comment by Meldiron** [comment]
The problem was addressed in 0.14 release 😇 Feel free to re-open issue if after upgrading you still encounter the problem.

**Comment by Infinil** [comment]
> The problem was addressed in 0.14 release 😇 Feel free to re-open issue if after upgrading you still encounter the problem.

I am currently 0.15.1 and I am facing the same issue. No idea why (using 0.18.4 CLI)

**Comment by LDY681** [comment]
> FYI, min value is -9223372036854775808 and max value is 9223372036854775807.
> 
> Something seems to be rounding the values to -9223372036854776000 and 9223372036854776000….

I found the same issue with appwrite console. I am currently on the v1.3.4.
After I created an attribute w/o specifying min and max and edited again.
These two fields were filled with -9223372036854776000 and 9223372036854776000.
Thus causing an error and prevent me to edit other attributes.

**Comment by sonicviz** [comment]
I'm trying to run through this Stripe tutorial and running into the same issue with the "created at" integer field.
https://dev.to/appwrite/start-selling-online-using-appwrite-and-stripe-3l04

                {
                    "key": "createdAt",
                    "type": "integer",
                    "status": "available",
                    "required": true,
                    "array": false,
                    "min": -9223372036854776000,
                    "max": 9223372036854776000,
                    "default": null
                }

✗ Error Invalid `max` param: Value must be a valid integer
PS E:\Source Control\AI Apps\Back End\Appwrite\almost-cookie-store2> appwrite deploy collection
? Which collections would you like to deploy? Orders (xxxx - orders)
ℹ Info Deploying collection Orders ( xxx- orders )
✓ Success Updated undefined (xxx )
ℹ Info Collection Orders ( orders ) already exists.
? Are you sure you want to override this collection? This can lead to loss of data! Type "YES" to confirm. YES
ℹ Info Deleting indexes and attributes ...
✗ Error Invalid `min` param: Value must be a valid integer
PS E:\Source Control\AI Apps\Back End\Appwrite\almost-cookie-store2> 

Any solution?

**Comment by sonicviz** [comment]
Update:
As pointed out above (and by Kenny in discord) "FYI, min value is -9223372036854775808 and max value is 9223372036854775807."

                {
                    "key": "createdAt",
                    "type": "integer",
                    "status": "available",
                    "required": true,
                    "array": false,
                    "min": -9223372036854775808,
                    "max": 9223372036854775807,
                    "default": null
                }

**Comment by tripolskypetr** [comment]
Still not fixed in appwrite 1.5

**Comment by tripolskypetr** [comment]
Thank god I new the auto generated cli will never work well and made ci/cd scripts

https://github.com/react-declarative/appwrite-backup-tool
